### PR TITLE
chore: minor rs translation changes

### PR DESCRIPTION
**Pull request by radulle**
Just some minor changes to Serbian translations.

### CODE REVIEW

File: app/views/devise/mailer/confirmation_instructions.rs.html.erb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <p>Zdravo!</p>
 
-<p>Možete potvrditi svoj email (<%= @email %>) prateći sledeću vezu:</p>
+<p>Možete potvrditi svoj email (<%= @email %>) prateći sledeći link:</p>
```
Comment (radulle) [inline app/views/devise/mailer/confirmation_instructions.rs.html.erb:3]: Veza is also ok, but link I believe is much more common.

File: app/views/devise/mailer/confirmation_instructions.rs.html.erb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-<p><%= link_to 'potvrdi Moj nalog', confirmation_url(@resource, confirmation_token: @token, host: 's95.rs') %></p>
+<p><%= link_to 'potvrdi moj nalog', confirmation_url(@resource, confirmation_token: @token, host: 's95.rs') %></p>
 
 <p>
-  Dobili ste ovu e-poštu jer je vaša adresa uneta prilikom registracije na veb lokaciji <%= link_to 'Sat 9am 5km', root_url(host: 's95.rs') %>.
+  Dobili ste ovaj e-mail jer je vaša adresa uneta prilikom registracije na web stranici <%= link_to 'Sat 9am 5km', root_url(host: 's95.rs') %>.
```
Comment (radulle) [inline app/views/devise/mailer/confirmation_instructions.rs.html.erb:8]: Web stranica is more common.

File: app/views/devise/mailer/email_changed.rs.html.erb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <p>Zdravo!</p>
 
 <% if @resource.try(:unconfirmed_email?) %>
-  <p>Vaš email je promenjen u <%= @resource.unconfirmed_email %>.</p>
+  <p>Vaš e-mail je promenjen u <%= @resource.unconfirmed_email %>.</p>
```
Comment (radulle) [inline app/views/devise/mailer/email_changed.rs.html.erb:4]: I changed this only for consistency with previous file.

File: app/views/devise/mailer/reset_password_instructions.rs.html.erb [hunk 4]
```diff
@@ -1,14 +1,14 @@
 <p>Zdravo <%= @resource.email %>!</p>
 
 <p>
-  Neko je zatražio vezu za promenu lozinke na veb lokaciji <%= link_to 'S95', root_url(host: 's95.rs') %>.
-  Da biste to uradili, idite na sledeću vezu:
+  Zatražili ste link za promenu lozinke na web stranici <%= link_to 'S95', root_url(host: 's95.rs') %>.
```
Comment (radulle) [inline app/views/devise/mailer/reset_password_instructions.rs.html.erb:4]: I believe it is "You requested link to for password change..."

**Review by vol1ura — APPROVED** [review]
Nikola, thank you very much!

**Pull request merged**